com.unity.render-pipelines.universal/ShaderLibrary/UnityGBuffer.hlsl

```diff
@@ -31,7 +36,7 @@ struct FragmentOutput
     half4 GBuffer1 : SV_Target1;
     half4 GBuffer2 : SV_Target2;
     half4 GBuffer3 : SV_Target3; // Camera color attachment
-    #if USE_SHADOWMASK
+    #if OUTPUT_SHADOWMASK
     half4 GBuffer4 : SV_Target4;
```

Comment:
Do we do any validation for platforms that don't support more than 4MRT?

Reply:
Yes. it will automatically goes back to forward renderer if the platform does not support the required number of RTs. QA already raised and tested it.
On a side note, the shadowMask add a 4th RT. RenderPass will add another RT in order to store depth-as-color. This means up to 6 RTs will be needed on mobiles.